code/stable/projectile/SRS/HTML/Projectile_SRS.html

```diff
@@ -459,6 +467,71 @@ <h2>Organization of Document</h2>
         </div>
       </div>
     </div>
+    <div id="Sec:GenSysDesc">
+      <div class="section">
+        <h1>General System Description</h1>
+        <p class="paragraph">
+          This section provides general information about the system. It identifies the interfaces between the system and its environment, describes the user characteristics, and lists the system constraints.
+        </p>
+        <div id="Sec:SysContext">
+          <div class="subsection">
+            <h2>System Context</h2>
+            <p class="paragraph">
+              <a href=#Figure:sysCtxDiag>Fig:sysCtxDiag</a> shows the system context. A circle represents an entity external to the software, the user in this case. A rectangle represents the software system itself (Projectile). Arrows are used to show the data flow between the system and its environment.
+            </p>
+            <div id="Figure:sysCtxDiag">
+              <figure>
+                <img src="../../../../datafiles/projectile/SystemContextFigure.png" alt="System Context" >
+                <figcaption>System Context</figcaption>
+              </figure>
+            </div>
+            <p class="paragraph">
+              The interaction between the product and the user is through an application programming interface. The responsibilities of the user and the system are as follows:
```

Comment:
I don't think this is true; the user interacts with Projectile through an input file, unless that is what is meant by "application programming interface".

Reply:
This is what is written accross other examples, We should create a different issue to fix this.

Reply:
Some other examples might _actually_ use an "application programming interface". Regardless, an issue is the best place to discuss this.